schemas/context/webinfo.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/webinfo",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Web Information",
+  "type": "object",
+  "description":
+    "Information about the web page and/or link of the ExperienceEvent.",
+  "definitions": {
+    "webinfo": {
+      "properties": {
+        "xdm:webPage": {
+          "title": "Web Page",
+          "$ref": "https://ns.adobe.com/xdm/context/webpage",
+          "description": "The web page information of the ExperienceEvent."
```

Comment:
Replace with something like:
This is the information about the page (URL) itself.